### DISCUSSION TITLE
Gaussian Splats

### DISCUSSION THREAD

**Discussion by gituser123456789000** [nagadomi/nunif#144] [General]
Check these out:
https://www.gmix.ai/
https://www.gmix.ai/more.html

https://huggingface.co/blog/gaussian-splatting
https://github.com/huggingface/blog/blob/main/gaussian-splatting.md

This is the next step.. next big, revolutionary feature needed... Start getting inside of photos and movies

**Comment by gituser123456789000** [comment]
These Gaussian Splats are probably more involved than I was expecting. Still something to look at, but I'm not sure we'll have something like this implemented any time soon.

**Comment by nagadomi** [comment]
I too believe that the evolution of 3D gaussian splatting will enable 2D image/video to be converted to VR180 interactive scene.
Once you have a 3D/4D gaussian splatting scene, you can simply render it with stereo camera settings. It should already be possible by building a photo set for gaussian splatting. (I have not tried it, so I do not know if there is any inconsistency between the left and right eye in practical use)
However, that is a completely different software from iw3.
